tools/go.sum

```diff
@@ -40,35 +40,45 @@ cloud.google.com/go/storage v1.8.0/go.mod h1:Wv1Oy7z6Yz3DshWRJFhqM/UCfaWIRTdp0RX
 cloud.google.com/go/storage v1.10.0/go.mod h1:FLPqc6j+Ki4BU591ie1oL6qBQGu2Bl/tZ9ullr3+Kg0=
```

Comment:
Weirdly I'm still getting checksum errors here. Could you try running `cd tools && rm go.sum && go get`?

Reply:
no changes to my go mod/sum files when i do that :/

Reply:
Huh weird, maybe it's on my end. I'll see if I can get another team member to double check 👍

Reply:
ok so... i ran a go mod tidy afterwards and it actually overwrote something the go get did... i checked in just the go get changes so see if that helps

Reply:
Still getting the error weirdly, I'll take a second look at this tomorrow

Reply:
https://github.com/butuzov/ireturn/issues/69

Reply:
pinned to v1.54.2